src/tests/unit/tests/background/stores/assessment-store.test.ts

```diff
@@ -1142,59 +1160,56 @@ describe('AssessmentStore', () => {
         await storeTester.testListenerToBeCalledOnce(initialState, finalState);
     });
 
-    test('onUpdateTargetTabId', async () => {
+    describe('onUpdateTargetTabId', () => {
         const tabId = 1000;
         const url = 'url';
         const title = 'title';
-        const tab: Tab = {
+        const tab = {
             id: tabId,
             url,
             title,
-        };
-        let onReject;
-        browserMock
-            .setup(b => b.getTab(tabId, It.isAny(), It.isAny()))
-            .returns((id, cb, reject) => {
-                onReject = reject;
-                cb(tab);
-            })
-            .verifiable();
-        const initialState = new AssessmentsStoreDataBuilder(
-            assessmentsProvider,
-            assessmentDataConverterMock.object,
-        ).build();
-        const finalState = new AssessmentsStoreDataBuilder(
-            assessmentsProvider,
-            assessmentDataConverterMock.object,
-        )
-            .withTargetTab(tabId, url, title)
-            .build();
+        } as chrome.tabs.Tab;
 
-        const storeTester =
-            createStoreTesterForAssessmentActions('updateTargetTabId').withActionParam(tabId);
-        await storeTester.testListenerToBeCalledOnce(initialState, finalState);
-        expect(() => onReject()).toThrowErrorMatchingSnapshot();
-    });
+        it.each([undefined, { tabId: 2000 }])('with persisted tab=%s', async persistedTab => {
```

Comment:
Adds coverage for branch in line 147 where this.state.persistedTabId is undefined